src/EditorFeatures/Test2/Rename/RenameTestHelpers.vb

```diff
@@ -22,13 +23,19 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
 
         <ThreadStatic>
         Friend _exportProvider As ExportProvider = MinimalTestExportProvider.CreateExportProvider(
-            TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic.WithParts(GetType(MockDocumentNavigationServiceFactory), GetType(RenameWaiter)))
+            TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic.WithParts(
+                GetType(MockDocumentNavigationServiceFactory),
+                GetType(RenameWaiter),
+                GetType(InteractiveDocumentSupportsCodeFixService)))
 
         Friend ReadOnly Property ExportProvider As ExportProvider
             Get
                 If _exportProvider Is Nothing Then
                     _exportProvider = MinimalTestExportProvider.CreateExportProvider(
-                        TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic.WithParts(GetType(MockDocumentNavigationServiceFactory), GetType(RenameWaiter)))
+                        TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic.WithParts(
+                            GetType(MockDocumentNavigationServiceFactory),
+                            GetType(RenameWaiter),
+                            GetType(InteractiveDocumentSupportsCodeFixService)))
```

Comment:
I don't think we want all of these tests picking up the ```InteractiveDocumentSupportsCodeFixService```. You might make and optional ```ExportProvider``` parameter to ```CreateWorkspaceWithWaiter``` and have the ```RenameCommandDisabledInSubmission``` test (and other relevant ones) create and pass in their special version composed with the ```InteractiveDocumentSupportsCodeFixService```.